src/Test.UnitTests.Sarif.Multitool/RebaseUriCommandTests.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Text;
+using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Multitool;
+using Moq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Multitool
+{
+    public class RebaseUriCommandTests : FileDiffingUnitTests
+    {
+        private static ResourceExtractor Extractor = new ResourceExtractor(typeof(RebaseUriCommandTests));
+        private RebaseUriOptions options;
+
+        public RebaseUriCommandTests(ITestOutputHelper outputHelper) : base(outputHelper) { }
+
+        [Fact]
+        public void RebaseUriCommand_RebaseRunWithArtifacts()
+        {
+            string testFilePath = "RunWithArtifacts.sarif";
+
+            this.options = new RebaseUriOptions
+            {
+                BasePath = @"C:\vs\src\2\s\",
+                BasePathToken = "SRCROOT",
+                Inline = true,
+                SarifOutputVersion = SarifVersion.Current,
+                PrettyPrint = true
+            };
+
+            RunTest(testFilePath);
+        }
+
+        protected override string ConstructTestOutputFromInputResource(string testFilePath)
```

Comment:
>ConstructTestOutputFromInputResource [](start = 34, length = 36)

Right, so when you use this, the base class calls this overridden method and then executes the comparisons.